executor/write.go

```diff
@@ -89,7 +89,11 @@ func updateRecord(ctx sessionctx.Context, h int64, oldData, newData []types.Datu
 			modified[i] = true
 			// Rebase auto increment id if the field is changed.
 			if mysql.HasAutoIncrementFlag(col.Flag) {
-				if err = t.RebaseAutoID(ctx, newData[i].GetInt64(), true); err != nil {
+				recordID, err := getAutoRecordID(newData[i], &col.FieldType)
+				if err != nil {
+					return false, false, 0, errors.Trace(err)
```

Comment:
no need to trace?

Reply:
In branch release-2.1. All errors are be traced.